### DISCUSSION TITLE
EFI stub during Proxmox installation using IPXE 1.1.21

### DISCUSSION THREAD

**Discussion by EnikeevIlya** [ipxe/ipxe#1119] [Q&A] [answered]
Hello everyone.

I have IPXE version 1.1.21, including the latest commit from this repository.

Using IPXE, I am trying to install Proxmox.

The logic is that the script performs the download .iso image from the official proxmox website, http://download.proxmox.com/iso / , then linux26 and initrd files are extracted from this image and sent to the server. And from there we start working with the IPXE script, which looks like this:

```
kernel ${base-url}/${proxmox-version}/linux26
initrd ${base-url}/${proxmox-version}/initrd
imgargs linux26 console=tty0 console=ttyS0,115200b8 nomodeset ramdisk_size=1500000000 rw splash=verbose initrd=initrd vga=791 video=vesafb:ywrap,mtrr
boot
```

The same logic works for the Proxmox version: 6.4-1, 7.4-1, 8.0-2, 8.1-1, at the same time, there are no problems with version 6.4-1 and it passes the installation perfectly.

With the rest of the versions, I get the following message where the installation freezes:

![Screenshot_16](https://github.com/ipxe/ipxe/assets/95696127/e98df324-0f88-4e95-a73c-864d5b6cae69)

If I enter the above commands manually without the IPXE script, the error is the same.
At the same time, **if I enter the same commands manually on a bare IPXE version 1.0.0, THERE ARE NO PROBLEMS!**

From here, I conclude that there may be something wrong with IPXE 1.1.21, which I work with, but what? I found a commit that you made earlier, in which, it seems to me, this error has been fixed. Or am I wrong? https://github.com/ipxe/ipxe/commit/09e8a1540

Thank you in advance for any help on this issue.

**Comment by mcb30** [comment, marked as answer]
Check the output of `imgstat` immediately before `boot` to make sure that you don't have any other images present (e.g. a script, or a graphical background).

**Reply by EnikeevIlya** [reply]
Thanks a lot, the problem was really in the previously hung images.
